checkov/terraform/checks/graph_checks/aws/S3BucketPubliclyReadableViaACL.yaml

```diff
@@ -0,0 +1,30 @@
+metadata:
```

Comment:
This is not the same as `630d3779-d932-4fbf-9cce-6e8d793c6916`. There are a lot more conditions.